cluster/node-pools/master-default/userdata.yaml

```diff
@@ -864,3 +864,30 @@ write_files:
           qps: {{ .Cluster.ConfigItems.event_rate_limit_config_qps }}
           burst: {{ .Cluster.ConfigItems.event_rate_limit_config_burst }}
 {{ end }}
+
+  - owner: root:root
+    path: /etc/kube-flannel/net-conf.json
+    content: |
+      {
+        "Network": "10.2.0.0/15",
+        "Backend": {
+          "Type": "vxlan"
+        }
+      }
+
+  - owner: root:root
+    path: /etc/kubernetes/cni/net.d/10-flannel.conflist
+    content: |
+      {
+        "name": "podnet",
+        "cniVersion": "0.3.1",
+        "plugins": [
+          {
+            "type": "flannel",
+            "delegate": {
+            "isDefaultGateway": true,
```

Comment:
Let's fix the indentation for the final version.

Reply:
I've kept the indentation verbatim from the config-map, can you point out the problem?

Reply:
Maybe it's just here on GitHub but `isDefaultGateway` should be one level to the right.

Reply:
Oh. I didn't notice the bracket in the previous line. You're right. I'll fix that in the next commit. Thanks.

Reply:
Fixed.